config/routes.rb

```diff
@@ -34,6 +34,10 @@
 
       resources :projects, param: :string_key, except: [:new, :edit] do
         resources :publish_targets, param: :string_key, except: [:new, :edit], module: 'projects'
+        resources :field_sets,                          except: [:new, :edit], module: 'projects'
+        resources :enabled_dynamic_fields,
+                  only: [:show, :update], module: 'projects',
```

Comment:
Is `module: 'projects'` necessary here (and in the other two cases), since this `resources` definition is within `resources :projects`?

Reply:
I couldn't get the route to use the nested controller without using the `scope` parameter. There might be a more succinct way to configure this though.

Reply:
Maybe this: https://github.com/cul/ldpd-hyacinth/blob/master/config/routes.rb#L130-L133

Reply:
We can always revisit it later.  I'm good with keeping as is.